velox/exec/Exchange.cpp

```diff
@@ -52,6 +52,7 @@ Exchange::Exchange(
           operatorCtx_->driverCtx()->queryConfig(),
           serdeKind_)},
       processSplits_{operatorCtx_->driverCtx()->driverId == 0},
+      driverId_{driverCtx->driverId},
```

Comment:
nit: we can just fetch from operatorCtx_->driverCtx()->driverId and not necessary save a copy of it.

Reply:
Just wanted to safe a couple of extra dereferences :-)